common/decklist.cpp

```diff
@@ -500,32 +547,36 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
             isSideboard = true;
         }
 
+        if (line.trimmed().isEmpty())
+            continue; // The line was "    " instead of "\n"
```

Comment:
While testing, for some reason entries as "______" started appearing randomly, and this stops it. There were no underscores in the inputs, and it was interpreting extra blank lines as this format for some reason. This just prevents that issue.